### ISSUE TITLE
Import `keras` from the `tensorflow` namespace

### ISSUE THREAD

**Issue by dhruvmanila** [TheAlgorithms/Python#4370]
```console
❯ rg keras
machine_learning/lstm/lstm_prediction.py_tf
9:from keras.layers import LSTM, Dense
10:from keras.models import Sequential
```

And thus, removing the `keras` dependency from `requirements.txt`.

Ref:
- https://github.com/keras-team/keras#under-construction
- https://keras.io/guides/functional_api/#setup

**Comment by keshavgbpecdelhi** [comment]
Here I am if you think i can.

**Comment by vidnyani** [comment]
Hello , this is my first time to open source would you please guide me how to get started for this project?

**Comment by keshavgbpecdelhi** [comment]
+1

**Comment by Dblue44** [comment]
test

**Comment by ak14bbk30** [comment]
> Hello , this is my first time to open source would you please guide me how to get started for this project?

for me also,this is the first time.

**Comment by keshavgbpecdelhi** [comment]
+1

**Comment by cclauss** [comment]
Some of this was landed in #4422
